pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -280,7 +285,9 @@ func RunUpdate(cmd *cobra.Command, args []string) {
 		}
 
 		noobaaAccount.Spec.DefaultResource = newDefaultResource
-		noobaaAccount.Spec.ForceMd5Etag = forceMd5Etag
+		if forceMd5EtagPtr != nil {
```

Comment:
@dannyzaken with this condition added, the code is working as expected.

Also in this case we need not to have force_md5_etag as *bool we can have it as bool only and update the code accordingly, Pls correct me if I am wrong here